<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>marketClass</class>
 <widget class="QWidget" name="marketClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>market</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>338</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/login/image/login/background_a.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="passWordLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>168</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/password.png);</string>
    </property>
    <property name="text">
     <string>密     码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passWordEdit">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>169</y>
      <width>164</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/input_box_a.png);
font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="userIDEdit">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>123</y>
      <width>164</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/input_box_a.png);
font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/username.png);</string>
    </property>
    <property name="text">
     <string>账  号：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>133</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/login/image/login/land.png);
}

QPushButton:hover{
border-image: url(:/login/image/login/landing.png);
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="narrowBtn">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>0</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/login/image/login/j.png);
}

QPushButton:hover{
border-image: url(:/login/image/login/j_a.png);
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/login/image/login/x.png);
}

QPushButton:hover{
border-image: url(:/login/image/login/x_a.png);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="stuBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>用户登录</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="adminBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>290</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>用户注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="market.qrc"/>
 </resources>
 <connections/>
</ui>
